### PR TITLE
OPGOPS-2762 Allow wordpress to upload larger files

### PR DESCRIPTION
**Pull request by carinadigital**

### CODE REVIEW

File: opg-wordpress-1604/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+RUN cd /tmp && \
+    wget --no-verbose https://en-gb.wordpress.org/latest-en_GB.tar.gz && \
+    tar -xf latest-en_GB.tar.gz && \
+    rm -rf wordpress/wp-content/themes/* && \
+    rm -rf wordpress/wp-content/plugins/hello.php wordpress/wp-content/plugins/akismet && \
+    mv wordpress/* /data/. && \
+    rm -rf wordpress latest-en_GB.tar.gz
+
+# Add wordpress configuration templates
+ADD docker/confd /etc/confd
```
Comment (damacus) [inline opg-wordpress-1604/Dockerfile]: It's recommended using COPY over ADD. 
https://nickjanetakis.com/blog/docker-tip-2-the-difference-between-copy-and-add-in-a-dockerile

File: opg-wordpress-1604/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+USER root
+
+# Remove files included from php-fpm container
+RUN rm -rf /app/* && \
+    ln -s /data /app/public
+
+ADD  docker/my_init.d /etc/my_init.d
```
Comment (damacus) [inline opg-wordpress-1604/Dockerfile]: The pattern in the base-1604 container is a little nicer. 

Single `COPY docker/ /`

File: opg-wordpress-1604/tests/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+       - WORDPRESS_AUTH_AUTH_KEY=WORDPRESS_AUTH_AUTH_KEY
+       - WORDPRESS_AUTH_NONCE_SALT=WORDPRESS_AUTH_NONCE_SALT
+       - WORDPRESS_AUTH_LOGGED_IN_KEY=WORDPRESS_AUTH_LOGGED_IN_KEY
+       - WORDPRESS_AUTH_NONCE_KEY=WORDPRESS_AUTH_NONCE_KEY
+       - WORDPRESS_AUTH_SECURE_AUTH_KEY=WORDPRESS_AUTH_SECURE_AUTH_KEY
+       - WORDPRESS_DATABASE_PASSWORD=wordpress
+       - WORDPRESS_AUTH_LOGGED_IN_SALT=WORDPRESS_AUTH_LOGGED_IN_SALT
+       - OPG_NGINX_SERVER_NAMES=localhost wordpress 127.0.0.1
+       - OPG_NGINX_SSL_FORCE_REDIRECT=True
+       - OPG_NGINX_CLIENT_MAX_BODY_SIZE=64M
```
Comment (damacus) [inline opg-wordpress-1604/tests/docker-compose.yml]: This seems like a lot of default values? 
```
WORDPRESS_DATABASE_PORT << default
OPG_NGINX_SERVER_NAMES << we could default this
OPG_NGINX_SSL_FORCE_REDIRECT << sensible default?
OPG_NGINX_CLIENT_MAX_BODY_SIZE << needed to make wordpress function like we want
```

Comment (damacus) [reply opg-wordpress-1604/tests/docker-compose.yml]: As per conversation, lets do this at a later date.

File: opg-php-fpm-1604/docker/confd/templates/app.conf.tmpl [hunk 4]
```diff
@@ -14,6 +14,12 @@ server {
     server_name {{ getv "/opg/nginx/server/names" }};
     {{ end }}
 
+    {{ if exists "/opg/nginx/client/max/body/size" }}
```
Comment (carinadigital) [inline opg-php-fpm-1604/docker/confd/templates/app.conf.tmpl]: php-pfm image overwrites the app.conf templated, so it's need here too. Now all images based on either base image will work with POST size

**Pull request merged**